packages/react-native-editor/CHANGELOG.md

```diff
@@ -11,8 +11,11 @@ For each user feature we should also add a importance categorization label  to i
 
 ## Unreleased
 
+## 1.80.0
+
```

Comment:
There's an extra space here compared to previous versions, can we remove it? Thanks!

Reply:
Good catch, I've gone ahead to remove that in https://github.com/WordPress/gutenberg/pull/42587/commits/fd7d1b21782787e18ff5a825d2b74203be77334d :)